pyproject.toml

```diff
@@ -200,5 +203,5 @@ exclude = [
 ]
 
 [build-system]
-requires = ["hatchling"]
+requires = ["hatchling", "hatch-vcs"]
```

Comment:
Why do we need `hatch-vcs`? Can't we just continue having the version on `pyproject.toml` and avoid an extra dependency?

It is also possible to have dynamic version [like this](https://github.com/sigmavirus24/github3.py/blob/main/pyproject.toml#L53) with hatch alone.

Reply:
I thought it's cooler and easier to maintain.
But if you prefer the `regex` method, I can change it back.

Reply:
It is OK, let's try!